README.md

```diff
@@ -217,7 +217,7 @@ class Car
   def dealer_options(*args); new; end
   notifies :design, :safety
   # adding additional event payload options for any instance method(s), by passing a hash as the last argument
-  notifies :dealer_options, { sport_package: true }
+  notifies [:dealer_options, { sport_package: true }]
```

Comment:
Is this change necessary?  If so, it would be a breaking change, and I'd like to avoid that.

Reply:
Actually it's not necessary - I had changed around the iteration logic slightly in `lib/debug_logging/class_notifier.rb` as I had thought it was necessary as part of the fix for the duplicate logging issue but it's not so I reverted it back to the original (similar to `/lib/debug_logging/class_logger.rb`).  I updated the `README.md` file accordingly.